python/orca/src/bigdl/orca/learn/pytorch/pytorch_ray_estimator.py

```diff
@@ -167,11 +168,15 @@ def __init__(
 
             head_worker = self.remote_workers[0]
             address = ray.get(head_worker.setup_address.remote())
+            driver_ip = ray.get(head_worker.get_node_ip.remote())
+            tcp_port = ray.get(head_worker.get_free_port.remote())
```

Comment:
In worker,  since we may not need `setup_address` any more, we could change it to `get_ip_port`, which directly returns `ip` and `port`.
Then we could use `driver_ip, tcp_port = ray.get(head_worker.get_ip_port.remote())`